runner/runner.go

```diff
@@ -316,8 +316,19 @@ func (r *Runner) prepareInputPaths() {
 }
 
 func (r *Runner) prepareInput() {
-	// check if file has been provided
 	var numHosts int
+	// check if a string input has been provided
+	if r.options.InputTargetHost != "" {
```

Comment:
We should use the logic from `goflags` that automatically splits the input into a slice. See `StringSliceVarP`

Reply:
You could check how it has been done in other projects of ours (e.g. `nuclei` or `dnsx`)